widgets-bundle/getting-started/creating-a-widget.md

```diff
@@ -160,3 +131,17 @@ function get_style_name($instance) {
 	return 'my-widget-styles';
 }
 ```
+
+## Widget banner image
+To use a custom image for the banner in the Plugins > SiteOrigin Widgets list you can either place it in a folder named `assets` and name the file `banner.svg` or you can use the `siteorigin_widgets_widget_banner` filter hook. The following code can be found in the example main widget file `my-awesome-widget.php` outside of the class declaration. If you put the code somewhere else, make sure to adjust the file path accordingly.
```

Comment:
Two commas added by Grammarly for this paragraph, if you're ok to make the change in this PR it'll be:

To use a custom image for the banner in the Plugins > SiteOrigin Widgets list, you can either place it in a folder named `assets` and name the file `banner.svg`, or you can use the `siteorigin_widgets_widget_banner` filter hook. The following code can be found in the example main widget file `my-awesome-widget.php` outside of the class declaration. If you put the code somewhere else, make sure to adjust the file path accordingly.